apps/web/src/routes/AuthLayoutRouter/components/ContextMenu/ContextMenu.jsx

```diff
@@ -128,7 +128,9 @@ export default function ContextMenu (props) {
 
   const homeOnClick = () => {
     if (window.location.pathname === rootPath) {
+      // TODO REDESIGN: but we dont go to the stream here?
```

Comment:
I don't think this is used by the new context menu. Were you stripping out the old stuff in this PR or saving it for another?

Reply:
ill do it separately